digid_eherkenning/models/certificates.py

```diff
@@ -147,9 +162,24 @@ class Meta:
                 violation_error_message=_(
                     "This configuration and certificate combination already exists."
                 ),
-            ),
+            )
         ]
 
+    def clean(self):
```

Comment:
please respect the order of the model methods, see https://docs.djangoproject.com/en/stable/internals/contributing/writing-code/coding-style/#model-style